easy_ViTPose/inference.py

```diff
@@ -268,14 +269,15 @@ def inference(self, img: np.ndarray) -> dict[typing.Any, typing.Any]:
             # Transform keypoints to original image
             keypoints[:, :2] += bbox[:2][::-1] - [top_pad, left_pad]
             frame_keypoints[id] = keypoints
+            scores_bbox[id] = scores[id] # Replace this with avg_keypoint_conf*person_obj_conf. For now, only person_obj_conf from yolo is being used.
 
         if self.save_state:
             self._img = img
             self._yolo_res = results
             self._tracker_res = (bboxes, ids, scores)
             self._keypoints = frame_keypoints
 
-        return frame_keypoints
```

Comment:
Here just return the keypoints, and on line 272 save the `scores_bbox` when `self.save_state`.